### PR TITLE
[ODS-5995] Added SingleTenant example

### PR DESCRIPTION
**Pull request by semalaiappan**

### CODE REVIEW

File: Compose/SingleTenant/bootstrap.sh [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+#!/bin/sh
+
+set -e
+set +x
+
+if [[ -z "$POSTGRES_PORT" ]]; then
+  export POSTGRES_PORT=5432
+fi
+
+EDFI_ODS_CONNECTION_STRING="host=$ODS_POSTGRES_HOST;port=$ODS_PGBOUNCER_PORT;username=$POSTGRES_USER;password=$POSTGRES_PASSWORD;database=EdFi_Ods;application name=EdFi.Ods.WebApi;"
```
Comment (simpat-jesus) [inline Compose/SingleTenant/bootstrap.sh:15]: Based on today daily, this script should only insert the OdsInstance, and leave the ApiClient config to the users

**Review by simpat-jesus — CHANGES_REQUESTED** [review]

File: Compose/pgsql/compose-single-tenant-env.yml [hunk 2]
```diff
@@ -26,9 +26,11 @@ services:
     environment:
       POSTGRES_USER: "${POSTGRES_USER}"
       POSTGRES_PASSWORD: "${POSTGRES_PASSWORD}"
-      API_MODE: "SharedInstance"
+      ODS_PGBOUNCER_PORT: "${PGBOUNCER_LISTEN_PORT:-6432}"
+      ODS_POSTGRES_HOST: pb-ods
     volumes:
       - vol-db-admin:/var/lib/postgresql/data
+      - ../SingleTenant/bootstrap.sh:/docker-entrypoint-initdb.d/bootstrap.sh   # Mount the bootstrap.sh script
```
Comment (simpat-jesus) [inline Compose/pgsql/compose-single-tenant-env.yml:33]: rename to docker-entrypoint-initdb.d/02-bootstrap.sh so we make sure any other script already included in the image is executed first

File: Compose/SingleTenant/bootstrap.sh (hunk 1 above)
Comment (semalaiappan) [reply Compose/SingleTenant/bootstrap.sh:15]: Updated

File: Compose/pgsql/compose-single-tenant-env.yml (hunk 2 above)
Comment (semalaiappan) [reply Compose/pgsql/compose-single-tenant-env.yml]: Updated

**Review by simpat-jesus — APPROVED** [review]

**Pull request merged**